src/render-html/css.js

```diff
@@ -149,6 +149,10 @@ th, td {
   white-space: nowrap;
   background: #ccc;
 }
+.mentioned {
+  padding-top: 4px;
```

Comment:
Hmm? Random padding?

Reply:
Na, without the padding it wasn't looking good, as for other messages the white blends nicely!